packages/client/components/ActionMeeting.tsx

```diff
@@ -18,22 +18,22 @@ interface Props {
 }
 
 const phaseLookup = {
-  [NewMeetingPhaseTypeEnum.checkin]: lazyPreload(() =>
+  checkin: lazyPreload(() =>
     import(/* webpackChunkName: 'NewMeetingCheckIn' */ './NewMeetingCheckIn')
   ),
-  [NewMeetingPhaseTypeEnum.updates]: lazyPreload(() =>
+  updates: lazyPreload(() =>
     import(/* webpackChunkName: 'ActionMeetingUpdates' */ './ActionMeetingUpdates')
   ),
-  [NewMeetingPhaseTypeEnum.firstcall]: lazyPreload(() =>
+  firstcall: lazyPreload(() =>
     import(/* webpackChunkName: 'ActionMeetingFirstCall' */ './ActionMeetingFirstCall')
   ),
-  [NewMeetingPhaseTypeEnum.agendaitems]: lazyPreload(() =>
+  agendaitems: lazyPreload(() =>
     import(/* webpackChunkName: 'ActionMeetingAgendaItems' */ './ActionMeetingAgendaItems')
   ),
-  [NewMeetingPhaseTypeEnum.lastcall]: lazyPreload(() =>
+  lastcall: lazyPreload(() =>
     import(/* webpackChunkName: 'ActionMeetingLastCall' */ './ActionMeetingLastCall')
   )
-}
+} as Record<NewMeetingPhaseTypeEnum, LazyExoticPreload<any>>
```

Comment:
Thanks for the taking the time to review this PR! The reason that I've used `as Record<...` instead of `as const` is we get type safety on the key with `as Record<...`. 
In the `phaseLookup` object, if I change `agendaitems` to `agendaitemz` using `as Record<...`, TypeScript will complain whereas it won't with `as const`. 

`as const` is definitely prettier though. Do you think I should go ahead and change these to `as const`?